### DISCUSSION TITLE
Renovate started treating net.fabricmc:yarn "22w13oneblockatatime+build.1" as "v22" and is suggesting it as an upgrade

### DISCUSSION THREAD

**Discussion by LunNova** [renovatebot/renovate#18580] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work. The unwanted upgrade started being suggested over the last 2 days.

### Describe the bug

Renovate started treating "22w13oneblockatatime+build.1" as "v22", and suggesting an upgrade.

It generated this upgrade PR: https://github.com/MinimallyCorrect/Mapping/pull/93

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Dependency net.fabricmc:yarn is part of group all non-major dependencies
DEBUG: Using group branchName template
DEBUG: Dependency net.fabricmc:yarn is part of group yarn115
DEBUG: Using group branchName template
DEBUG: Dependency net.fabricmc:intermediary is part of group all non-major dependencies
DEBUG: 6 flattened updates found: net.fabricmc:intermediary, net.fabricmc:intermediary, net.fabricmc:yarn, net.fabricmc:yarn, net.fabricmc:intermediary, net.fabricmc:intermediary
DEBUG: Returning 4 branch(es)


DEBUG: recreateClosed is false(branch="renovate/major-yarn115")
DEBUG: findPr(renovate/major-yarn115, Update dependency net.fabricmc:yarn to v22, !open)(branch="renovate/major-yarn115")
DEBUG: prAlreadyExisted=false(branch="renovate/major-yarn115")
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by Churro** [comment, marked as answer]
Since PR https://github.com/renovatebot/renovate/pull/18469 renovate also identifies dependencies with classifiers and thereby found an update for `net.fabricmc:yarn:1.15.2+build.9:v2`. This is not a bug, renovate just did what it is supposed to do.

**Comment by rarkins** [comment]
I accidentally commented in the linked PR instead of this issue/discussion. Here's a screenshot:

![image](https://user-images.githubusercontent.com/6311784/198032515-b944f078-ba81-4d50-870e-82ec2d0eaace.png)
